src/commands/remoteDebug/remoteDebugCommon.ts

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { SiteConfigResource } from 'azure-arm-website/lib/models';
+import * as opn from 'opn';
+import * as vscode from 'vscode';
+import { SiteClient } from 'vscode-azureappservice';
+import { DialogResponses } from 'vscode-azureextensionui';
+import { ext } from '../../extensionVariables';
+
+export function reportMessage(message: string, progress: vscode.Progress<{}>): void {
+    ext.outputChannel.appendLine(message);
+    progress.report({ message: message });
+}
+
+export function checkForRemoteDebugSupport(siteConfig: SiteConfigResource): void {
+    // So far only node on linux is supported
+    if (!siteConfig.linuxFxVersion.toLowerCase().startsWith('node')) {
```

Comment:
Missed one of my previous comments:

> This will get a null ref exception on windows. Suggest:
> ```
> if (siteConfig.linuxFxVersion && !siteConfig.linuxFxVersion.toLowerCase().startsWith('node'))
> ```